server.js

```diff
@@ -267,33 +267,41 @@ app.post('/slambook', async (req, res) => {
 //   }
 // });
 
```

Comment:
It is recommended to use the option { new: true } when updating a document with findByIdAndUpdate to return the updated document. Consider adding a comment to explain the purpose of this option.